generators/_init-web/templates/plain/.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,67 @@
+## Release notes
+
+*List of addressed issues and PRs since the last release*
+
+
+## Checklists
+
+### Release preparation
+
+* [x] Create new `release-x.x.x` branch (based on `develop` branch)
+* [ ] Collect changes and write [release notes](#release-notes)
+* [ ] Draft release PR in GitHub that merges the `release-x.x.x` into the `master` branch
+
+### Release dependencies first
+
+In case of dependent Phovea/TDP repositories follow [dependency tree](https://wiki.datavisyn.io/phovea/fundamentals/development-process#dependency-hierarchy) from the top:
+
+* [ ] Release dependent repositories if they contain changes first before proceeding here
+* [ ] Replace git dependencies in *package.json* with new version range (e.g., `"phovea_core": "^2.3.1"` when published on npm **or** `"phovea_core": "github:datavisyn/tdp_core#semver:^8.0.0"` for non-published repositories)
+* [ ] Commit and push new dependencies
+* [ ] Wait until build is successful
+* [ ] Repeat with other repositories/dependencies or proceed with next section
+
+### Update version
+
+* [ ] Check version numbers of dependencies again
+* [ ] Check if build is successful
+* [ ] Update this version number following [semver](https://semver.org)
+* [ ] Commit and push *package.json* with new version number
+* [ ] Wait until build is successful
```

Comment:
After increasing the version number waiting until the build is successful is not necessary, because the build will succeed / fail regardless of the version number

```suggestion
```